src/util/util.go

```diff
@@ -42,33 +42,34 @@ func ReadFile(path string) *os.File {
 	return file
 }
 
-//BytesToMB transforms bytes into megabytes
+// BytesToMB transforms bytes into megabytes
 func BytesToMB(sizeBytes int64) float64 {
 	return float64(sizeBytes) / 1024. / 1024.
 }
 
-//MBToBytes transforms megabytes into bytes
+// MBToBytes transforms megabytes into bytes
 func MBToBytes(sizeMB float64) int64 {
 	return int64(sizeMB) * 1024 * 1024
 }
 
-//IntegerMin returns the minimum of two integers
+// IntegerMin returns the minimum of two integers
 func IntegerMin(x, y int) int {
 	if x < y {
 		return x
 	}
 	return y
 }
 
-//RunCommandAndLog runs a command in the terminal, logs the result and returns it
+// RunCommandAndLog runs a command in the terminal, logs the result and returns it
 func RunCommandAndLog(cmd *exec.Cmd) string {
 	var out bytes.Buffer
 	var stderr bytes.Buffer
 	cmd.Stdout = &out
 	cmd.Stderr = &stderr
 	err := cmd.Run()
 	if err != nil {
-		log.Fatalf("%s: %s", fmt.Sprint(err.Error()), stderr.String())
+		log.Infof("Command result: %s", out.String())
```

Comment:
I added this line earlier when debugging because I had issues with seeing what the exact issue and error message was with the Serverless deployment, not sure if we wish to leave this here

Reply:
actually, I'd like to keep it there for future debugging.